subsys/bluetooth/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_library(subsys__bluetooth INTERFACE)
```

Comment:
I believe so.

My intention is to do this when I implement the TODO: "Introduce more CMake extentions to reduce boilerplate and enforce conventions in driver/ subsys/ ext/ CMakeLists.txt code."